docker_setup_scripts/install_rust.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -euo pipefail -x
+
+readonly RUST_VERSION=1.78.0
+
+install_dependencies() {
+  if [ -f /etc/os-release ]; then
+    . /etc/os-release
+    case "$ID" in
+      amzn|rhel|centos|fedora)
```

Comment:
From tests, it looks like almalinux is missing from this list.

Reply:
Thanks.. just fixed it.